README.md

```diff
@@ -47,9 +47,9 @@ import Ember from 'ember';
 import { CanMixin } from 'ember-can';
 
 export default Ember.Route.extend(CanMixin, {
-  beforeModel: function() {
+  beforeModel() {
     if (!this.can('write post')) {
```

Comment:
What if super depends on a value being returned? Do we assign the result to a var and return if can't write?